src/FAQ/Faq.js

```diff
@@ -1,14 +1,38 @@
 import React from 'react'
 import faqData from './FaqData'
+
+const Link = props =>
+    <a className="color-navigation font-weight-bold" target="blank" href={props.href}>
+        {props.children}
+    </a>
+
+const AnswerWithLinks = props =>
+    props.data.map((data, index) => {
+        let anser = <span>{data}</span>
+        if (data === 'Hier') {
+            anser = <a
+                className="color-navigation font-weight-bold"
+                href={index === 0 ? props.tutorialLink : props.clickvideoLink}
+                target="blank">
+                {data}
+            </a>
+        }
+        return anser
+    })
+
+
 const Faq = () => (
-    <div className='row mt-4'>
+    <div className='row my-4'>
         <div className='col-xl-9 col-12' >
             <ul className="list-group">
-                {faqData.map((e,i) =>
-                    <li className="list-group-item" key={i}>
-                        <div className='font-weight-bold'><span >F : </span>{e.question}</div>
-                        <div className='mt-1'><span className='font-weight-bold'>A : </span><span>{e.answer}</span> <span>{e.email}</span></div>
-                        {e.link}
```

Comment:
dieser Teil hier, wenn die FAQ-Objekte unten links als Attribut haben (in etwa, da musst du noch die `<a>`Tags hinzufügen und bei email im link href dann noch `mailto:`): 

```
<span className='font-weight-bold'>A : </span>
<span>{e.answer}</span> 
<span className="color-navigation font-weight-bold">{e.email}</span>
{ e.links.map(link) =>  
	<span className="color-navigation font-weight-bold">{e.link}</span>}
```

-> wenn du unten keinen Spezialfall einbaust, brauchst du hier oben auch viel weniger Änderungen! `AnswerWithLinks` wird komplett überflüssig, genauso wie `Link` .